meta/netbeansrelease.json

```diff
@@ -586,6 +586,35 @@
             "year": "2021"
         }
     },
+    "release1263": {
+        "position": "16",
+        "ant": "ant_latest",
+        "jdk": "jdk_1.8_latest",
+        "jdk_apidoc": "https://docs.oracle.com/javase/8/docs/api/",
+        "maven": "maven_3.3.9",
+        "versionName": "12.6.301",
+        "tlp": "true",
+        "apidocurl": "https://bits.netbeans.org/12.6/javadoc",
```

Comment:
I've just started [run 807](https://ci-builds.apache.org/job/Netbeans/job/netbeans-vscode/807/) with [parameter METAURL](https://ci-builds.apache.org/job/Netbeans/job/netbeans-vscode/807/parameters/) set to `https://raw.githubusercontent.com/apache/netbeans-jenkins-lib/0f4c351e89e1945ae440f53d0ac7c2ba9c0acae4/meta/netbeansrelease.json`. According to the [log output](https://ci-builds.apache.org/job/Netbeans/job/netbeans-vscode/807/consoleFull) the branding info was picked properly up:
```
[releasejson] Writing releasinfo file /home/jenkins/jenkins-agent/workspace/Netbeans/netbeans-vscode/nbbuild/build/netbeansrelease.xml
[releasejson] Branding computed info: metabuild.DistributionURL=https://netbeans.apache.org/nb/updates/12.6/updates.xml.gz?{$netbeans.hash.code}
[releasejson] Branding computed info: metabuild.PluginPortalURL=https://netbeans.apache.org/nb/plugins/12.6/catalog.xml.gz
[releasejson] Branding computed info: metabuild.RawVersion=12.6.301
[releasejson] Branding computed info: metabuild.apachepreviousversion=9.0,10,11.0,11.1,11.2,11.3,12.0,12.1,12.2,12.3,12.4,12.5,12.5.301,12.6
[releasejson] Branding computed info: metabuild.ComputedSplashVersion=12.6.301
[releasejson] Branding computed info: metabuild.ComputedTitleVersion=12.6.301
[releasejson] Branding computed info: metabuild.logcli=
```

Reply:
Would it work with the following PR https://github.com/apache/netbeans/pull/3443 applied ? I did'nt remember the blocker on pre 12.4 vscode. If it pass it should not break the upcoming vscode release.

Reply:
@JaroslavTulach yes, it's expected to be right for VSNetBeans. Although you can see that `metabuild.apachepreviousversion` is incorrect both here and in NetBeans IDE. Not a major concern at the moment as it doesn't break anything, but it demonstrates the inaccurate picture of release history.